feature/matching-service/common/src/main/java/com/pob/umpa/feature/matching_service/common/enums/LessonStatus.kt

```
@@ -0,0 +1,8 @@
+package com.pob.umpa.feature.matching_service.common.enums
+
+enum class LessonStatus(
+
+) {
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)